packages/analytics-core/src/plugins/destination.ts

```diff
@@ -213,6 +215,12 @@ export class Destination implements DestinationPlugin {
 
   handleSuccessResponse(res: SuccessResponse, list: Context[]) {
     this.fulfillRequest(list, res.statusCode, SUCCESS_MESSAGE);
+    const dropEventSet = list.reduce(function (filtered, context) {
```

Comment:
CASE 2: when events have been sent out successfully.

Reply:
how about changing `dropEventSet` to `sentEventSet`? As they are removed from the storage not because of being dropped but being sent successfully.

Reply:
it's not only the event that has been successfully sent. It's also including the event which missing some info. I think `filterEventSet` make more sense?